src/styles/blocks/card.css

```diff
@@ -0,0 +1,110 @@
+:root {
+  --mainFontFamily: "Roboto", sans-serif;
+  --darkColor: #060b35;
+  --mediumColor: #616070;
+  --lightColor: white;
+  --highlightColor: #00acdc;
+  --borderColor: #f3f3f3;
+}
+
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid var(--borderColor);
+  border-radius: 5px;
+
+  background-color: var(--lightColor);
+  font-family: var(--mainFontFamily);
+}
+
+.product-card__image {
+  align-self: center;
+
+  height: 134px;
+  width: 160px;
```

Comment:
So we stretch the image?

Reply:
Probably not, but tbh I don't understand it . Should I wrap it in something or the problem is with it being an inline element?

Can you please throw some documentation  at me? 😞

Reply:
@ulositska, why should we manipulate its size at all?

Reply:
@silchyn I've probably messed up somewhere else in the code. Because if I don't set size to the picture, it overflows its parent element the card itself 😕 
https://prnt.sc/JMD42knjXQTQ

Reply:
I mean I can put it in a container, and set its size instead, It won't be hurting the picture dimensions this way, but I amn't sure if it achieves that much more(